doc/getting-started.md

```diff
@@ -88,6 +88,26 @@ val spark = SparkSession
 val sc = spark.sparkContext  
 ```
 
+The SparkSession can be configured with SSL enabled:
+
+```scala
+val spark = SparkSession
+  .builder()
+  .appName("myApp")
+  .master("local[*]")
+  .config("spark.redis.host", "localhost")
+  .config("spark.redis.port", "6379")
+  .config("spark.redis.auth", "passwd")
+  .config("spark.redis.ssl", "true")
+  .getOrCreate()
+  
+val sc = spark.sparkContext  
+```
+
+```bash
+java -Djavax.net.ssl.trustStorePassword=password -Djavax.net.ssl.trustStore=path/to/keystore -Djavax.net.ssl.trustStoreType=jceks ...
```

Comment:
Not sure we should include this line into the doc. It's only needed if someone starts Redis with a generated CA certificate. And in that case if they submit an app to spark cluster, they will need to configure java options in spark specific way, i.e. using `spark.executor.extraJavaOptions` and `spark.driver.extraJavaOptions` (https://spark.apache.org/docs/latest/configuration.html). Maybe we should create a separate doc page describing it since it's more advanced topic (not for a getting-started page). What do you think?

Reply:
sounds like a good idea.